common-codegen/build.gradle

```diff
@@ -1,6 +1,41 @@
+plugins {
+    id "net.ltgt.apt-idea" version "0.15"
```

Comment:
How does this plugin work? After importing the gradle project with idea I still had to run gradle to generate the missing code.

Reply:
I used this to configure annotation processor generated folders..., still apparently during build its mentioned that with the new gradle version its recommended to use the annotationProcessor directive, but for me it didn't work properly, so i let that as a convenience until i fix it later, for idea i disabled the annotation processor.